boot/seal_test.go

```diff
@@ -87,6 +87,25 @@ func mockNamedKernelSeedSnap(rev snap.Revision, name string) *seed.Snap {
 }
 
 func mockGadgetSeedSnap(c *C, files [][]string) *seed.Snap {
+	mockGadgetYaml := `
+volumes:
+  volumename:
+    bootloader: grub
+`
+	if files == nil {
+		files = [][]string{}
+	}
```

Comment:
I don't think this is neccessary? An empty or nil should act the same, especially for append()
```suggestion
```